app/models/llm_model.rb

```diff
@@ -20,4 +65,7 @@ def tokenizer_class
 #  updated_at        :datetime         not null
 #  url               :string
 #  api_key           :string
+#  bot_username      :string
```

Comment:
we should not store usernames here, we should store bot_user_id, we need to be robust to renames. given we already have user_id we should remove then?

Reply:
Sure, let's do it. I just wanted to make sure bot usernames stays consistent when deleting/creating by toggling the enabled attr. Maybe it was too much.